internal/deviations/deviations.go

```diff
@@ -12,11 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package deviations documents and controls vendor deviation
-// workarounds that exist in the featureprofiles test suite.
+// Package deviations documents defines arguments to enable
+// workarounds for a featureprofiles test suite.
 //
-// This is a temporary workaround and will be deprecated in the
-// future.
+// Deviations are useful to temporarily enable passing for other dependent
```

Comment:
I'm not sure what "enable passing for other dependent subcases" means.

Maybe "enable tests to pass when a device is not OpenConfig compliant"?

Reply:
Would it be useful to include text like this:
`If a test case contains five 'subtests', subtests 2-5 may be dependent on subtest 1.  If a device does not yet implement subcase 1, subcases 2-5 may not be able to pass.  With a deviation for subcase 1, a device can demonstrate passing for subcases 2-5 and non-compliance for subcase 1.   This enhances visibility.`



I would like to avoid suggesting that introducing a deviation is useful to make a test pass as that can be interpreted  very broadly.

Reply:
How about this? "Deviations are useful to temporarily bypass non-compliant failure conditions using workarounds so further testing could be done."

This wording avoids any mentioning of enabling or passing.

Reply:
Done